pulsar-broker/src/main/java/com/yahoo/pulsar/broker/loadbalance/BrokerHostUsage.java

```diff
@@ -71,25 +91,152 @@ public BrokerHostUsage(PulsarService pulsar) {
      *
      * @throws IOException
      */
-    public String getBrokerHostUsage() throws IOException {
-        StringWriter writer = new StringWriter();
+    public synchronized SystemResourceUsage getBrokerHostUsage() throws IOException {
```

Comment:
If calculations are done every 1min, but the data es retreived on a greater period, does it really make sense to make calculations at shorter periods? Would we always return the average usage of the last minute?